extensions/istio/model-v1alpha3/pom.xml

```diff
@@ -62,6 +62,11 @@
       <groupId>org.projectlombok</groupId>
       <artifactId>lombok</artifactId>
     </dependency>
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter-engine</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Is the explicit dependency still needed after removing the `mvn -o` flag?

Reply:
No, It doesn't require it anymore. I've removed it.

Reply:
OK, please update again, since branch is stale after merging the extension fixes